src/windowmanagement/TopMenuBar.tsx

```diff
@@ -25,22 +54,63 @@ export function TopMenuBar({
         backgroundColor: 'var(--mantine-color-gray-filled)'
       }}
     >
-      <Button size={'xs'} color={'gray'}>
-        Asset
-      </Button>
+      <Menu
+        position={'bottom-start'}
+        menuItemTabIndex={0}
+        offset={4}
+        withArrow
+        arrowPosition={'center'}
```

Comment:
All the `Menu`s have these same 5 props. This resulting in quite some repetitive code, and if we decide to change one there are many places that need changed. Wondering if there is some better way to do it... Maybe through Matine

My first thought would be to put the props in a object and three spread them on each component, but not sure I like that a lot either... Maybe making a new component?  

Or, maybe we could discuss how we want to deal with this type of situation more generally, since it will happen again :)